### PR TITLE
Improve CLA message

### PR DESCRIPTION
**Pull request by mosabua**

### CODE REVIEW

**Review by nineinchnick — APPROVED** [review]

**Review by hashhar — COMMENTED** [review]
While we're doing this it might be an improvement to set expectations (i.e. CLAs may take some time to be processed and where to reach out in case it's been over <n> days (n being 7 in my opinion).)

**Comment by mosabua** [comment]
That works for me. Same email would be where to reach out... or on slack in #dev ?

**Comment by mosabua** [comment]
Any thoughts @martint ?

**Comment by mosabua** [comment]
Done now @hashhar

File: .clabot [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
   "contributors": "https://api.github.com/repos/trinodb/cla/contents/contributors",
-  "message": "Thank you for your pull request and welcome to our community. We require contributors to sign our [Contributor License Agreement](https://github.com/trinodb/cla/raw/master/Trino%20Foundation%20Individual%20CLA.pdf), and we don't seem to have you on file. In order for us to review and merge your code, please submit the signed CLA to cla@trino.io. For more information, see https://github.com/trinodb/cla.",
+  "message": "Thank you for your pull request and welcome to the Trino community. We require contributors to sign our [Contributor License Agreement](https://github.com/trinodb/cla/raw/master/Trino%20Foundation%20Individual%20CLA.pdf), and we don't seem to have you on file. Continue to work with us on the review and improvements in this PR, and submit the signed CLA to cla@trino.io. Processing takes five to ten business days on average. The CLA needs to be on file before we merge your changes. For more information, see https://github.com/trinodb/cla",
```
Comment (martint) [inline .clabot]: I would just say "processing may take a few days".

Comment (mosabua) [reply .clabot]: Done.

**Comment by mosabua** [comment]
Thanks @martint

**Pull request merged**